images/kamu-web-ui/rootfs/etc/nginx/conf.d/default.conf

```diff
@@ -0,0 +1,12 @@
+server {
+    listen       8080;
+    server_name  localhost;
+
+    gzip         on;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html;
+        try_files $uri $uri/ /index.html;
```

Comment:
What happened to our other configuration that used to be in `nginx.conf`?

Things like:
```
keepalive_timeout 65;
sendfile on;
```

Reply:
Looking inside the image itself looks like these parameters exist in `/etc/nginx/nginx.conf` which includes this file in `conf.d/` ... but we actually flattened the config on purpose:

It's much easier to see the entire config in git rather than try to guess what additional configuration is defined in the base image, or run into issues when base image config changes between versions.

IMO `conf.d` has no use in containerized env, as this `nginx` will always be running only one service, so flat config is much nicer.

Reply:
In this case using separate unprivileged image gives almost zero benefit, because 95% of it still has to be replicated and maintained by us, namely the configuration file, which is exactly what has been done in #241